dotcom-rendering/docs/values/README.md

```diff
@@ -0,0 +1,13 @@
+# Values
+
+## Approachable
+
+The project should have as few barriers as possible for anyone to work on it, including designers, UX, people outside of the Guardian and more.
+
+### Lines in the sand
+
+#### `.env` shouldn't be required
```

Comment:
+1 An important point not to be forgotten

Reply:
Do you think it's worth enforcing in some or other way? That might be how we access to config is the only way I can think of.